db/types.go

```diff
@@ -208,6 +213,30 @@ type OrgMember struct {
 	UpdatedAt      time.Time      `json:"updatedAt" bson:"updatedAt"`
 }
 
+// OrgMemberAuthFields defines the fields that can be used for member authentication.
+type OrgMemberAuthField string
+
+const (
+	OrgMemberAuthFieldsName         OrgMemberAuthField = "name"
+	OrgMemberAuthFieldsSurname      OrgMemberAuthField = "surname"
+	OrgMemberAuthFieldsMemberNumber OrgMemberAuthField = "memberNumber"
+	OrgMemberAuthFieldsNationalID   OrgMemberAuthField = "nationalID"
+	OrgMemberAuthFieldsBirthDate    OrgMemberAuthField = "birthDate"
+	OrgMemberAuthFieldsEmail        OrgMemberAuthField = "email"
+	OrgMemberAuthFieldsPhone        OrgMemberAuthField = "phone"
+)
+
+type OrgMemberAuthFields []OrgMemberAuthField
+
+type OrgMemberAggregationResults struct {
+	// MemberIDs is a list of member IDs that are result of the aggregation
+	Members []primitive.ObjectID `json:"memberIds" bson:"memberIds"`
+	// Duplicates is a list of member IDs that were found to be duplicates in string
+	Duplicates []primitive.ObjectID `json:"duplicates" bson:"duplicates"`
+	// Empties is a list of member IDs that had colums were found to be empty in string
+	Empties []primitive.ObjectID `json:"empties" bson:"empties"`
```

Comment:
Empties sounds strange to me.

maybe MissingData? (or WithMissingData??)
`// MissingData is a list of member IDs that had empty fields`